pyproject.toml

```diff
@@ -1,46 +1,40 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "sodym"
 version = "0.1.0"
-description = ""
-authors = ["Jakob Dürrwächter, Merlin Hosak, Stefan Pauliuk"]
+authors = [
+    {name = "Jakob Dürrwächter"},
+    {name = "Merlin Hosak"},
```

Comment:
```suggestion
    {name = "Sally Dacie"},
    {name = "Merlin Hosak"},
```

Reply:
Do you want to add your email as well @JakobBD ? I didn't want to add it for you...

Reply:
I'm fine with just the name, thanks for asking! :-)